### ISSUE TITLE
Gemma 1b-it finetune worked great for multi-turn chat, but failed for `dialect text → standard text` conversion

### ISSUE THREAD

**Issue by hasin023** [huggingface/huggingface-gemma-recipes#26]
I’m fine-tuning **Gemma 1B instruction-tuned** locally and ran into a failure I can’t explain.

### Target task

Normalize **regional dialect text** into **Standard Text** so downstream LLMs / rule-based extractors don’t hallucinate, while extracting data from the regional text (since now it'll be extracting from the standard text)

### What worked

I previously fine-tuned Gemma 1B it for a **multi-turn phone survey agent** using the standard chat template:

* `<start_of_turn>user / model`
* Instruction-heavy system prompt
* Multi-turn conversational data

**Result:** Tuned model followed instructions extremely well and performed reliably.

### What I changed

I reused **the same fine-tuning script, base model, trainer, and hyperparams**, but switched to a **single-turn parallel text task**:

```
User:
Convert the following dialect text to standard text.
Respond ONLY with the converted text.
[dialect text]
Model:
[standard text]
```

Dataset = `dialect_text → standard_text`.
And, still using Gemma 1B **instruction-tuned** as the base.

### Result

The fine-tuned model performed **very poorly**:

* Inconsistent outputs
* Often ignored the instruction
* Much worse than the multi-turn chat model

### What I’m trying to understand

Where is the conceptual mistake?

* Is dialect text → standard text fundamentally a **translation / seq2seq task**, not an instruction-following task?
* Does instruction-tuned Gemma fight against clean text-to-text mapping?
* Is this translation task different on an architecture level from the basic LLM architecture?
* Should this be trained without LLM fine-tuning & rather moved to a different type of ML model?
* Why does the *harder* multi-turn task succeed, but the “simpler” rewrite task fail?

And I **_apologize_** in advance if I come of as rude, But I’m not looking for `use a bigger model` answers— I want to understand **why this failed at a training-objective level**. Maybe I am conceptually doing some mistake? Maybe I don't know something that I need to understand on an architecture level about this task of mine?

I’d really appreciate your insights on this matter
